### DISCUSSION TITLE
"brew services list" returning error [solved]

### DISCUSSION THREAD

**Discussion by h-archer** [Homebrew/discussions#1494] [Everyday usage]
Hello, Can anyone give me possible reason(s) why `brew services list` is giving the following error?

```shell
Error: Failure while executing; `/bin/launchctl\ list\ \'homebrew.mxcl.dnsmasq\'` exited with 113. Here's the output:
```
I just upgraded brew. (`brew upgrade`)

Brew 3.1.8
macOS 10.13 (no errors/warnings from `brew doctor` other that not supporting macos 10.13)
php@7.3
Valet 2.15.0
nginx 1.19.10
Dnsmasq 2.85

**Comment by carlocab** [comment]
Hard to say. What happens when you run
```
/bin/launchctl list 'homebrew.mxcl.dnsmasq`
```
directly?

**Reply by h-archer** [reply]
It goes into a command-like input with "> "

Just like when you enter mysql mode for example. And now I cannot get out of it :)

Ok I existed with ctrl+c

**Comment by SMillerDev** [comment]
> macOS 10.13 (no errors/warnings from brew doctor)

That's impossible, brew will complain about the macOS version

**Reply by h-archer** [reply]
@SMillerDev Yes that warning is there about not supporting macos 10.13. But other than that no other messages.

**Comment by h-archer** [comment]
This is getting strange. After installing php@7.3, now running `brew services list` gives:
```shell
Error: Failure while executing; `/bin/launchctl\ list\ \'homebrew.mxcl.php@7.3\'` exited with 113. Here's the output:
```
Before it was complaining about `homebrew.mxcl.dnsmasq` (post above), now it's saying `homebrew.mxcl.php@7.3`.

Does this reveal anything??

**Reply by SMillerDev** [reply]
What happens when you run 
```
/bin/launchctl list 'homebrew.mxcl.php@7.3'
```

**Comment by SMillerDev** [comment]
This has been fixed in the latest `brew services` commit

**Reply by h-archer** [reply]
@SMillerDev Thanks!
